client/src/components/pages/welcome.tsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import Login from "../elements/Login";
 import Signup from "../elements/Signup";
-const welcome = () => {
+import "../../css/welcome.css";
```

Comment:
file doesn't exist

Reply:
forgot to include it should be there now

Reply:
ah ok lemme try again